hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/compact/CompactUtil.java

```diff
@@ -195,7 +195,8 @@ public static void verifyFieldClassIsCompactSerializable(Class<?> fieldClass, Cl
                 + "cannot be serialized with zero configuration Compact "
                 + "serialization because this type is not supported yet. If you "
                 + "want to serialize '" + clazz + "' which uses this class in "
-                + "its fields, consider writing a CompactSerializer for it.");
+                + "its fields, consider writing a CompactSerializer for the main class or "
```

Comment:
```suggestion
                + "its fields, consider writing a CompactSerializer for either the '" + clazz + "' or the '" + fieldClass + "'."
```
I believe this would suffice here, as we talk about which one is the main class or which one is the field class

Reply:
ok applied